projects/g++/Menge/CMakeLists.txt

```diff
@@ -19,7 +19,8 @@ INCLUDE_DIRECTORIES (${MENGE_SRC_DIR}/MengeCore/Agents)
 INCLUDE_DIRECTORIES (${MENGE_SRC_DIR}/MengeCore/Runtime)
 INCLUDE_DIRECTORIES (${MENGE_SRC_DIR}/MengeCore/PluginEngine)
 
-set( CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} -fpermissive" )
+set( CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} -fpermissive -std=c++11" )
```

Comment:
this means you don't use compilers without c++11 support. While, this is nice and should be ok, you may still want to check this in the cmakefile.